src/operator/groupBy.ts

```diff
@@ -240,6 +240,8 @@ class GroupDurationSubscriber<K, T> extends Subscriber<T> {
       group.error(err);
     }
     this.parent.removeGroup(this.key);
+    this.unsubscribe();
+    this.parent.remove(this);
```

Comment:
wouldn't it be better `GroupBySubscriber::removeGroup` manages subscription as well when it's specified to remove?